src/mpid/ch4/src/ch4_impl.h

```diff
@@ -617,6 +617,17 @@ static inline int MPIDI_CH4I_valid_group_rank(MPIR_Comm * comm, int rank, MPIR_G
         }                                                               \
     } while (0)
 
+/* Generic routine for checking synchronization at every RMA operation.*/
+#define MPIDI_CH4U_RMA_OP_CHECK_SYNC(target_rank, win)                                 \
```

Comment:
This is just for refactoring (=no logic change), right?

Reply:
Yes.